alerting/src/main/kotlin/org/opensearch/alerting/DocumentLevelMonitorRunner.kt

```diff
@@ -290,7 +290,8 @@ object DocumentLevelMonitorRunner : MonitorRunner() {
                 listOf(it.first),
                 listOf(it.second),
                 triggerCtx,
-                monitorResult.alertError() ?: triggerResult.alertError()
+                monitorResult.alertError() ?: triggerResult.alertError(),
+                workflowExecutionId // TODO add execution id for other types of monitors also
```

Comment:
Do we want to use workflowExecutionId or executionId? I thought we might want to extend to make this more generic, so would that be better to do it now?